### PR TITLE
created appbar & navbar with responsive

### PR DESCRIPTION
**Pull request by elvisnguy**
## Change Description 👋

Fixed userprofile header

## Demo 📺
image
image
image
image
image

### CODE REVIEW

**Comment by elvisnguy** [comment]
## Change Description 👋

I created MainContent to show content & use Nested Route to link to navbar's option each other. 

## Demo 📺

https://www.loom.com/share/c7c0292167374f6a9ba3a7586b0093a6

**Comment by elvisnguy** [comment]
Change Description 👋
I created "<" button to close and open navbar, fixed button "company" in navbar. All dashboard look like this now. I'll try finish responsive tomorrow.

Demo 📺
image

**Comment by elvisnguy** [comment]
Change Description 👋
Fixed gerenal dashboard layout and button color on tablet, mobile.

Demo 📺
image
image
image
image

File: src/Dashboard/components/MainContent.vue [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  </div>
+</template>
+
+<script>
+export default {};
+</script>
+
+<style>
+.container {
+  padding: 5% 0% 0% 5%;
```
Comment (truonghungit) [inline src/Dashboard/components/MainContent.vue:15]: use pixel or rem instead % for padding, margin

**Review by truonghungit — APPROVED** [review]
Good job Thành,

File: src/Dashboard/components/NavBar.vue [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  text-decoration: none;
+}
+.move-btn {
+  position: absolute;
+  top: 100px;
+  right: -20px;
+}
+#company-btn {
+  width: 70px !important;
+  height: 50px !important;
```
[the remark this answers is not in the document]
Comment (elvisnguy) [reply src/Dashboard/components/NavBar.vue:111]: let me try sir

File: src/Dashboard/components/MainContent.vue (hunk 1 above)
Comment (elvisnguy) [reply src/Dashboard/components/MainContent.vue:15]: I got it

**Review by halfangel37 — COMMENTED** [review]
I didnt see any big problem now but you can delete unecessary comment code, and I see you use a lot of !important , is it really necessary to use  ?

**Comment by elvisnguy** [comment]
> I didnt see any big problem now but you can delete unecessary comment code, and I see you use a lot of !important , is it really necessary to use ?
> 
> ```
> 
> ```

It's all necessary to comment and use !important bro, because some css property of vue we can't access so I used it

**Pull request merged**